.gitlab-ci.yml

```diff
@@ -9,6 +9,7 @@ stages:
   - dogfood
   - release-build
   - release-publish
+  - post
```

Comment:
💡 Tho it is very tempting to name this stage `notify` - with that name it will simply not work 🤷‍♂️. According to my tests, no jobs from stage named `notify` appear in GH PR list. Maybe some internal working of DD GitLab is causing the problem here - I don't know.

Reply:
Ah! So that's my problem! Thanks!